src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -15,8 +15,67 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
+    public static final String USERGUIDE_URL = "https://ay2122s2-cs2103t-t12-3.github.io/tp/UserGuide.html";
+    public static final String NEWLINE_AND_INDENTATION ="\n     ";
+    public static final String HELP_MESSAGE = "Tracey supports the following features: Find, Add, Delete, Edit, Clear, "
+        + "Summarise, List, Help and Filter.\n"
+        + "This guide aims to showcase the syntax used for the aforementioned features.\n"
+        + "For a full comprehensive guide, please refer to " + USERGUIDE_URL
+        + "\n"
+
+        + "\n1. FIND a particular contact in Tracey."
+        + NEWLINE_AND_INDENTATION
+        + "Format: find NAME"
+        + NEWLINE_AND_INDENTATION
+        + "Example 1: find John\n"
+
+        + "\n2. ADD a student with relevant details into Tracey."
```

Comment:
Maybe can provide the user with the list of possible tags

Reply:
Noted! I think this is a good idea. I have added this in! 👍